src/charm.py

```diff
@@ -55,9 +55,11 @@ def __init__(self, *args):
         self._kratos_relation_name = "kratos-endpoint-info"
         self._prometheus_scrape_relation_name = "metrics-endpoint"
         self._loki_push_api_relation_name = "logging"
-        self._login_ui_service_command = "identity_platform_login_ui"
-        self._log_dir = Path("/var/log")
-        self._log_path = self._log_dir / "login_ui.log"
+        self._login_ui_service_command = "/usr/bin/identity-platform-login-ui"
+        self._log_dir = "/var/log"
+        self._log_path = f"{self._log_dir}/ui.log"
+
+        assert Path(self._log_dir).is_dir()
```

Comment:
I think it's better to avoid using asserts in non test code. AFAIK this will only be run when unit tests run, so it does not help. 

Finally, you want to know that this path is a directory in the `workload` container whereas this code runs on the `charm` container (you need to do it through pebble).

Reply:
For point 1 that's me being out of the python loop, I recall that in normal conditions that runs as expected (not sure it was good practice though), happy to change to a try/except

Will change it to run it via pebble on workload container as that was actually what I was trying to achieve, my bad